addons/mail/static/src/core/common/message.xml

```diff
@@ -63,6 +63,7 @@
                         </div>
                         <div
                             class="position-relative d-flex"
+                            name="message content"
```

Comment:
We often don't pay too much attention to that, but technically (in HTML) name is not a valid attribute for `div` elements.

Maybe we should have a specific class instead, or for custom data we should use `data-` something.